Web Docs - Stepper Component (#2727)
hashicorp/design-system: website/docs/components/stepper/list/partials/accessibility/accessibility.md

## Conformance rating

<Doc::Badge @type="success">Conformant</Doc::Badge>

When used as recommended, there should not be any WCAG conformance issues with this component.

## Best Practices

### Using processing status

> Only use the `@status` value of `processing` on a step, when it is related to a loading process, or if user interaction can not continue until completed. If used outside this use case, this state status is not conformant.

Reviewer: A couple of things confused me reading through this, what do you think?

```suggestion
Only use the `@status` value of `processing` on a step, when it is related to a loading or background process, or if user interaction can not continue until completed. If used outside these use cases this status is not conformant.
```

Reviewer 2: This sounds more clear to me. Implemented this suggestion.

> Only use the `@status` value of `processing` on a step, when it is related to a loading or background process, or if user interaction can not continue until completed. If used outside these use cases this status is not conformant.

Reviewer 3: ```suggestion
The `@status` value of "processing" should only be used on a step in relation to a loading or background process or if user interaction is halted until a process completes. Using this status outside these use cases is not conformant.
```
